sdcm/fill_db_data.py

```diff
@@ -2976,6 +2983,7 @@ def _get_table_name_from_query(query):
         match = regexp.search(query)
         if match:
             return match.groupdict()["table_name"]
+        return None
```

Comment:
Returning "None" is default behavior. Why do you add this line?

Reply:
it is fixing pylint issue, moved to separate commit